DigitalExcellence/dex-backend
# Add Postman tests to pipeline (#189)

## Description

- Renamed the action name to `run-tests`
- Added a new job to run the Postman tests in parallel to the existing job for running the unit tests when opening a new PR against `develop`

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Open a new PR against `develop`
2. You should see the new GitHub action running automatically

## Link to issue

Closes: #189
